Jenkinsfile

```diff
@@ -119,7 +127,10 @@ stage("Check dependent projects against updated schema") {
           [$class: 'StringParameterValue',
             name: 'SCHEMA_BRANCH',
             value: env.BRANCH_NAME],
-        ]
+          [$class: 'StringParameterValue',
+            name: 'SCHEMA_COMMIT',
+            value: env.GIT_COMMIT]
+        ], wait: wait
```

Comment:
Yes, this is the opposite side from the govuk-puppet change that allowed the downstream builds to report to Github individually. Could this timing also go into the buildProject job?